jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java

```diff
@@ -58,10 +58,7 @@ private BlockMgrCache(int readSlots, int writeSlots, final BlockMgr blockMgr)
     {
```

Comment:
Cache0 is the `Optional` of caches :-)

`BlockMgrCache` is mainly used for 32-bit TDB only - 64-bit uses memory mapped file and does not layer a `BlockMgrCache` on top.